engine/title.asm

```diff
@@ -1,19 +1,648 @@
+	call TitleSeq_IncreaseJumpTableIndex
+	ld a, $01
+	ldh [hBGMapMode], a
+	ret
+
+TitleSeq_HoOh:: 							; 60f6 
+	call Set_HoOh
+	ld a, $10
+	ld [wJumptableIndex + 2], a
+	call TitleSeq_IncreaseJumpTableIndex
+	ld a, $01
+	ldh [hBGMapMode], a
+	ret
+
+TitleSeq_PressButtonInit:: 				; 6106 
+	ld hl, wJumptableIndex
+	inc [hl]
+	ld hl, wJumptableIndex + 2
+	ld de, DecodeNybble0Table - 3	; DecodeNybble0Table - 3 = $0C00 
+	ld [hl], e
+	inc hl
+	ld [hl], d
+	ret	
+	
+TitleSeq_TitleScreenInputAndTimeout:: 	; 6114 
+	ld hl, wJumptableIndex + 2
+	ld e, [hl]
+	inc hl
+	ld d, [hl]
+	ld a, e
+	or d
+	jr z, .psbtn_reset
+	dec de
+	ld [hl], d
+	dec hl
+	ld [hl], e
+	call GetJoypad
+	ld hl, hJoyState			; hJoyState = $FFA3 
+	ld a, [hl]
+	and D_UP | B_BUTTON | SELECT						; UP + B + SELECT brings you to the SRAM clear screen. 
+	cp D_UP | B_BUTTON | SELECT
+	jr z, .psbtn_sramclear
+	ld a, [hl]
+	and SELECT						; SELECT will bring you to the debug menu. 
+	jr nz, .psbtn_gotodebug
+	ld a, [hl]
+	and $09
+	ret z
+	
+.psbtn_play
+	ld a, $00					; MainMenu 
+	jr .psbtn_nextseq
+
+.psbtn_gotodebug
+if DEBUG 
+	ld a, $01					; DebugMenu 
+	jr .psbtn_nextseq
+else
+	ret
+endc
+
+.psbtn_sramclear
+	ld a, $02					; SRAMClearMenu 
+	
+.psbtn_nextseq
+	ld [wJumptableIndex + 1], a
+	ld hl, wJumptableIndex
+	set 7, [hl]
+	ret
+
+.psbtn_reset
+	ld hl, wJumptableIndex
+	inc [hl]
+	xor a
+	ld [wMusicFadeID], a 			; C1A7
+	ld [wMusicFadeID + 1], a
+	ld hl, wMusicFade 			; C1A5 
+	ld [hl], 8
+	ret	
+	
+TitleSeq_FadeMusicOut:: 					; 615C 
+	ld a, [wMusicFade]
+	and a
+	ret nz
+	ld a, 3
+	ld [wJumptableIndex + 1], a
+	ld hl, wJumptableIndex
+	set 7, [hl]
+	ret	
+
+SetLYOverrides:: 				; 616C 
+	ld hl, wLYOverrides
+	ld c, $30
+.setly_loop
+	ld [hli], a
+	dec c
+	jr nz, .setly_loop
+	ret
+
+PrintPMSubtitle:: 				; 6176 
+	coord hl, 2, 6
+	ld b, 15
+	ld a, $69
+	jr LoadPrintArea 			; 6186 
+
+PrintVersion:: 					; 617f 
+	coord hl, 4, 1
+	ld b, $09
+	ld a, $60
+	
+LoadPrintArea:: 				; 6186 
+	ld [hli], a
+	inc a
+	dec b
+	jr nz, LoadPrintArea
+	ret
+
+PrintPMJapaneseChara:: 			; 618C 
+	coord hl, 15, 2
+	ld a, "こ"
+	lb bc, 4, 4
+	jr PrintBoxArea
+
+PrintPokemonLogo:: 				; 6196 
+	coord hl, 15, 3
+	ld [hl], $B8
+	coord hl, 15, 4
+	ld [hl], $B9
+	coord hl, 1, 2
+	ld a, $80
+	ld bc, $0E04
+	
+PrintBoxArea::
+	ld de, SCREEN_WIDTH
+	push bc
+	push hl
+	
+.xloop
+	ld [hli], a
+	inc a
+	dec b
+	jr nz, .xloop
+	pop hl
+	add hl, de
+	pop bc
+	dec c
+	jr nz, PrintBoxArea
+	ret
+	
+PrintCopyRight::
+	coord hl, 3, 17
+	ld a, $41
+	ld b, $0D
+	
+.loop
+	ld [hli], a
+	inc a
+	dec b
+	jr nz, .loop
+	ret
+	
+SRAMClearMenu:: 				; 61C6 
+	call ClearTileMap
+	call GetMemSGBLayout
+	call LoadFont
+	call LoadFontExtra
+	ld hl, SRAMClear_TextMsg1
+	call PrintText 	; E70 
+	ld hl, SRAMClear_WinPOS
+	call CopyMenuHeader 		; 1D50
+	call VerticalMenu 			; 1D87
+	jp c, Init
+	ld a, [wMenuCursorY]				; CC2A
+	cp $01
+	jp z, Init
+
+	callfar InitAllSRAMBanks, $05
+	jp Init
+
+SRAMClear_TextMsg1::
```

Comment:
formatting should be TAB + db + space + byte as in:

``
SRAMClear_TextChoice::
	db %11000000
``